src/scheduler/data_types.h

```diff
@@ -176,6 +177,19 @@ typedef sch_resource_t usage_t;
 typedef void event_ptr_t;
 typedef int (*event_func_t)(event_ptr_t*, void *);
 
+typedef struct sched_cmd {
+	int cmd;   /* sched command */
+	char *jid; /* jobid assisiated with cmd if any else NULL */
+} sched_cmd_t;
+
+typedef struct svr {
+	char *svrhost;
```

Comment:
We don't need port here as pbs_connect/pbs_connect_extend, needs `host:port` or `host` (for dflt port) or `NULL` (for current host with dflt port) syntax.

Reply:
I am actually talking about the Server's port which is useful in future. Since this svr_t is meant for server's info better to have the port on which Server is using.

Reply:
@suresh-thelkar I understand you point, but we use pbs_connect() where we need `host[:port]` syntax, and I am not able to collect any use case for server port for now, if we have any use case where we will use it in future I am happy to add it.